contracts/modules/common/BaseModule.sol

```diff
@@ -98,14 +98,30 @@ contract BaseModule is Module {
 
     /**
     * @dev Utility method enbaling anyone to recover ERC20 token sent to the
-    * module by mistake and transfer them to the Module Registry. 
+    * module by mistake and transfer them to the Module Registry.
     * @param _token The token to recover.
     */
     function recoverToken(address _token) external {
         uint total = ERC20(_token).balanceOf(address(this));
         ERC20(_token).transfer(address(registry), total);
     }
 
+    /**
+     * @dev Verify that the wallet is unlocked.
+     * @param _wallet The target wallet.
+     */
+     function verifyUnlocked(BaseWallet _wallet) internal view {
+        require(!guardianStorage.isLocked(_wallet), "MV2: wallet locked");
+     }
+
+    /**
+     * @dev Verify that the caller is the module or the wallet owner.
+     * @param _wallet The target wallet.
+     */
+    function verifyWalletOwner(BaseWallet _wallet) internal view {
+        require(msg.sender == address(this) || isOwner(_wallet, msg.sender), "BM: must be wallet owner");
+    }
+
```

Comment:
Why did you move the `verifyUnlocked ` and `verifyWalletOwner ` logic to internal methods? They seem to be used only once. It is more gas efficient?

Reply:
Yes, it saves quite a bit of gas on deployment. Modifiers are inlined in the code so calling an internal method within them reduces the amount of code duplicated in each function that uses the modifier.